### DISCUSSION TITLE
Monthly gem5 Developers' Meeting - November 9th 5pm (UTC)

### DISCUSSION THREAD

**Discussion by ivanaamit** [gem5/gem5#482] [gem5-dev]
[slides](https://github.com/gem5/gem5/files/13312452/2023-11-09.monthly.meeting.pdf)

If you would like to add anything to the agenda please create a comment below or reach out to Jason (@powerjg; [jason@lowepower.com](mailto:jason@lowepower.com)).

Date and time: November 9th, 9am Pacific *standard* time (note: this is the week after the time change in the US)

Agenda:
- gem5 status over the past month
  - Update on CI testing infrastructure
- Ongoing work (probably no need for discussion)
  - Classic prefetchers in Ruby #502 
  - Improvements to workloads/resources #532 #510
  - Kconfig #69 
- Specific discussions
  - x86 performance issues #398 @bgodala
  - Memory leak in Ruby #518 #514 #508 @giactra @mysoreanoop @mattsinc 
  - Failing weekly and compiler tests @BobbyRBruce 
  - Status of branch predictor updates @andysan @dhschall @bgodala
- Plans for gem5 v23.1
- Other items from the floor

If you're tagged on the agenda, your input would be appreciated during the meeting. Apologies if you get emails each time the agenda is updated.

**Comment by BobbyRBruce** [comment]
Here are copies of the slides shown during this meeting:

* [2023-11-08-monthly-meeting.pdf](https://github.com/gem5/gem5/files/13334414/2023-11-08-monthly-meeting.pdf)
* [2023-11-08-monthly-meeting.pptx](https://github.com/gem5/gem5/files/13334415/2023-11-08-monthly-meeting.pptx)

**Comment by BobbyRBruce** [comment]
A recording of this meeting has been uploaded to the gem5 YouTube Channel: https://youtu.be/ZzCJdtr1mv0?si=xijIMYQ2n2tUqwqf.
